data_sources/sysmon_for_linux_eventid_11.yml

```diff
@@ -85,6 +85,21 @@ fields:
 - timestartpos
 - user
 - vendor_product
+output_fields:
+- action 
+- dest 
+- file_access_time 
+- file_create_time 
+- file_hash 
+- file_modify_time 
+- file_name 
+- file_path 
+- file_acl 
+- file_size 
+- process_guid 
+- process_id 
+- user 
```

Comment:
can we add process_name, process_exec where avaialble?

Reply:
As discussed, we can't add it because it is not part of the CIM data model.